lib/demos/material/bottom_app_bar_demo.dart

```diff
@@ -165,21 +170,30 @@ class _DemoBottomAppBar extends StatelessWidget {
         data: IconThemeData(color: Theme.of(context).colorScheme.onPrimary),
         child: Row(
           children: [
-            IconButton(
-              tooltip: MaterialLocalizations.of(context).openAppDrawerTooltip,
-              icon: const Icon(Icons.menu),
-              onPressed: () {},
+            Semantics(
+              sortKey: const OrdinalSortKey(1),
```

Comment:
I'm curious if this works without 1, 2, and 3. For example, if you only add it above the row

```
Semantics(
 sortKey: const OrdinalSortKey(1),
 Row(
...
```

Reply:
The Row currently does not have its own SemanticNode, which makes its 3 buttons siblings to the fab. In order for this to work we need to add a SemanticNode (and focus) for the Row itself. It's just the focus might not look great when the fab is docked and the BottomAppBar has a notch.

Reply:
If you wrap the row in Semantics, can you avoid having three separate Semantics for its button children?

Reply:
Yes I think so. Lmk if you are leaning towards this

Reply:
If we can achieve this with a simpler solution (2 semantic nodes), let's go with that

Reply:
Sounds good! Updated.